contributors/devel/welcome-to-kubernetes-new-developer-guide.md

```diff
@@ -0,0 +1,126 @@
+
+Welcome to Kubernetes! (New Developer Guide)
+============================================
+
+2017-06-21 through 2017-08-06
+Original author: [Alexander Campbell](https://github.com/alexandercampbell)
```

Comment:
Just my opinion: original author information is point in time. I don't think it encourages/discourages anyone from participating and updating things. If you want to continue updating this document after it's merged, I think new contributors would be glad to have your contributions. It's really on all of us (the k8s community, and sig-contribex) to update these documents and keep them relevant, not on one person. Perhaps identifying the sig that owns it (like contribex)?